### DISCUSSION TITLE
velero kopia data uploader

### DISCUSSION THREAD

**Discussion by SCLogo** [vmware-tanzu/velero#8427] [Community support Q&A]
We use velero with kopis and datamover option. Our clusters use spot instances. Often when backup is in progress and the node where the data mover velero pod is running got a spot termination, then the pod get an evict command. After that velero/kopia does not try to continue the upload on a different node, just set the dataupload object to failed. 
Is there any option to retry the upload before set it as failed?

**Comment by kaovilai** [comment]
IIUC velero is not yet setup to retry anything when there's velero/nodeagent pod eviction.
